dds/DCPS/DomainParticipantImpl.cpp

```diff
@@ -1052,10 +1075,13 @@ DomainParticipantImpl::delete_contained_entities()
   if (disc)
     disc->fini_bit(this);
 
+  /*
```

Comment:
remove or add reason why it's commented-out

Reply:
It was a test that I never cleaned up, but I'm uncommenting it out because I think it should probably go back to how it was for now.